### PR TITLE
Fix OAuth2 multitenancy sample

### PR DESCRIPTION
**Pull request by eleftherias**
Fixes gh-6834

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@eleftherias Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-security&pullRequestId=6836)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-security&pullRequestId=6836) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@eleftherias Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-security&pullRequestId=6836)!

File: samples/boot/oauth2resourceserver-multitenancy/README.adoc [hunk 1]
```diff
@@ -23,13 +23,21 @@ By default, the tests are pointing at a mock Authorization Server instance.
...
 ```
 
-where "subject" is the value of the `sub` field in the JWT returned by the Authorization Server.
+where "subject" is the value of the `sub` field in the JWT sent in the `Authorization` header,
+
+or the phrase
+```bash
+Hello, subject for tenantTwo!
+```
+where "subject" is the value of the `sub` field in the response from the Authorization Server.
```
Comment (jzheaux) [inline samples/boot/oauth2resourceserver-multitenancy/README.adoc]: Could we add a comment here about it being introspection?

```suggestion
where "subject" is the value of the `sub` field in the Introspection response from the Authorization Server.
```

Comment (eleftherias) [reply samples/boot/oauth2resourceserver-multitenancy/README.adoc]: Done!

**Comment by jzheaux** [comment]
Thanks, @eleftherias, for the PR! This is now merged into `master`.

**Pull request merged**